posts_report.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+use Timber\Timber;
+
+Timber::render( [ 'posts_report.twig' ], [] );
+
+include dirname( __FILE__ ) . '/underscore_templates/posts-reports.tpl.php';
```

Comment:
The file is named `posts_reports.tpl.php` (with underscore), so it can not find it